src/tink/Anon.hx

```diff
@@ -20,6 +20,18 @@ class Anon {
       );
   }
 
+  public static macro function cascade<T>(exprs : Array<ExprOf<T>>) : Expr {
```

Comment:
cascade is going to be nearly identical to merge, except for the presence of a parameter telling mergeExpressions to permit field name overrides.